provider/resources.go

```diff
@@ -57,34 +58,16 @@ func makeResource(mod string, res string) tokens.Type {
 
 // Provider returns additional overlaid schema and metadata associated with the provider..
 func Provider() tfbridge.ProviderInfo {
-	dnsimpleProv := dnsimple.Provider()
-	p := shimv2.NewProvider(dnsimpleProv)
-
 	prov := tfbridge.ProviderInfo{
-		P:           p,
+		P:           pf.ShimProvider(shim.NewProvider()),
 		Name:        "dnsimple",
 		Description: "A Pulumi package for creating and managing dnsimple cloud resources.",
 		Keywords:    []string{"pulumi", "dnsimple"},
 		License:     "Apache-2.0",
 		Homepage:    "https://pulumi.io",
 		Repository:  "https://github.com/pulumi/pulumi-dnsimple",
 		Config:      map[string]*tfbridge.SchemaInfo{},
-		GitHubOrg:   "terraform-providers",
 		Resources: map[string]*tfbridge.ResourceInfo{
-			"dnsimple_record": {Tok: makeResource(mainMod, "Record"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"name": {
-						Type: "string",
-					},
-					"type": {
-						Type: makeType(mainMod, "RecordType"),
-					},
-				},
-				Docs: &tfbridge.DocInfo{
-					Markdown: []byte(" "),
-				},
-				DeprecationMessage: "This resource is deprecated.\nIt will be removed in the next major version.",
-			},
 			"dnsimple_domain":                   {Tok: makeResource(mainMod, "Domain")},
```

Comment:
I would take this opportunity to remove all the resource mappings, just to keep `resources.go` clean.